server/src/main/java/org/elasticsearch/cluster/metadata/IndexMetadata.java

```diff
@@ -947,7 +947,7 @@ public IndexMetadata withTimestampRanges(
         }
         IndexLongFieldRange allowedEventIngestedRange = eventIngestedRange;
         // remove this check when the EVENT_INGESTED_RANGE_IN_CLUSTER_STATE version is removed
-        if (minClusterTransportVersion.before(TransportVersions.EVENT_INGESTED_RANGE_IN_CLUSTER_STATE)) {
+        if (minClusterTransportVersion.before(TransportVersions.V_8_15_0)) {
```

Comment:
@henningandersen - Do this look safe to you for dealing with the mixed cluster situation around event.ingested in cluster state? If yes, I'm thinking we should leave this in and remove the comment above that says:
```
// remove this check when the EVENT_INGESTED_RANGE_IN_CLUSTER_STATE version is removed
```

Reply:
This change collapses versions which are not _distinctly_ used. Serverless is well past these versions, and release versions only know about specific versions, which is what is being converted here to versioned constants.

Note, @thecoop the comment above should be removed since the constant it refers to no longer exists.

Reply:
Yup, just waiting for confirmation on expectations, then I'll replace it with a `@UpdateForV9`

Reply:
This looks safe wrt event_ingested, assuming the mapping of previous constants to release versions is done correctly. Not sure how that is validated though, but that is besides the question.

Reply:
It's validated by all BwC continuing to work